test/e2e/nodefeature/nodefeature.go

```diff
@@ -104,6 +104,9 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	SidecarContainers = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("SidecarContainers"))
 
+	// Sig-node: add e2e tests for KEP-4191
+	SplitDisk = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("SplitDisk"))
```

Comment:
I wonder if we should keep the name aligned with the feature. So `SplitFilesytem`, perhaps?

Reply:
The kubelet feature is named `KubeletSeparateDiskGC`, assuming that the tests carry the same name is reasonable.

Reply:
Right. Makes sense.

Then I propose to use `SeparateDiskGC` instead of `SplitDisk`. How does it sound?

Reply:
@kwilczynski the string for `KubeletSeparateDiskGC` is also `"KubeletSeparateDiskGC"`. Why not use that?

Reply:
@saschagrunert, of course we can!

I assumed people want to omit the "Kubelet" part (prefix) from "KubeletSeparateDiskGC". But if that is OK to include, then by all means.

This will make things more consistent and easier to troubleshoot, since grep would pick it up, etc.

Reply:
I assume that the reason for the naming prefix "Kubelet" is inherited by the fact that the features are API Server as well as kubelet visible. So that it's clear for the end user that we speak about a kubelet feature.

Reply:
Probably. I was referring to the naming approach people used here in end-to-end tests per:

- https://github.com/bitoku/kubernetes/blob/splitfs/test/e2e/nodefeature/nodefeature.go#L25-L114

... which makes me wonder whether this is correct. Note the whitespace:

```go
	// TODO: document the feature (owning SIG, when to use this feature for a test)
	ImageID = framework.WithNodeFeature(framework.ValidNodeFeatures.Add(" ImageID"))
```

Reply:
I agree with renaming.
In that case, we have to change the test-infra as well.
https://github.com/kubernetes/test-infra/blob/630e7a28b79c2f50535b8d439c540587d5d5f259/config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml#L2843-L2852

Reply:
I made a PR in test-infra https://github.com/kubernetes/test-infra/pull/32943